CHANGELOG.md

```diff
@@ -34,6 +34,8 @@ Main (unreleased)
 - Add `metrics` prefix to the url of list instances endpoint (`GET /agent/api/v1/instances`) and 
   list targets endpoint (`GET /agent/api/v1/metrics/targets`). (@marctc)
 
+- Embed time zone data to enable Promtail pipelines using the `location` field on Windows machines. (@tpaschalis)
```

Comment:
```suggestion
- Embed timezone data to enable Promtail pipelines using the `location` field on Windows machines. (@tpaschalis)
```

Reply:
Ahh, a classic dilemma :D going forward with the suggestions ^^